appeals/api/src/server/endpoints/integrations/integrations.controller.js

```diff
@@ -9,9 +9,18 @@ import {
 	mapDocumentSubmission,
 	mapAppeal,
 	mapDocument
-} from './integrations.mapper.js';
+} from './integrations.mappers/index.js';
+
 import { produceAppealUpdate } from './integrations.service.js';
+import { createAuditTrail } from '#endpoints/audit-trails/audit-trails.service.js';
 import { EventType } from '@pins/event-client';
+import BackOfficeAppError from '#utils/app-error.js';
+
+const auditSettings = {
+	systemUserId: '00000000-0000-0000-0000-000000000000',
+	appellantCaseMsg: 'The Appellant case was received',
+	lpaQuestionnaireMsg: 'The LPA questionnaire was received'
+};
```

Comment:
Ideally these strings should be defined in the constants file (appeals/api/src/server/endpoints/constants.js)